src/site/content/en/blog/oyo-case-study/index.md

```diff
@@ -0,0 +1,182 @@
+so they decided to create
+[OYO Lite](https://play.google.com/store/apps/details?id=com.oyo.consumerlite),
+a TWA built on their existing PWA.
+
+Starting with a PWA is essential.
+Users expect a native-like experience in an app they run from the Android launcher,
+so web content served in a TWA must provide that experience, including:
+- Fast load and response times
+- Reliability when the user has limited or no connectivity
+- A unified look and feel (by providing, for example, a splash screen and app color)
+
+{% Aside %}
+Meeting the [Lighthouse requirements for a PWA](/lighthouse-pwa)
+is actually a prerequisite for TWAs.
+To learn more about building a PWA,
+see the [Installable collection](/installable).
+{% endAside %}
+
+If you already have a PWA,
+the steps for creating a basic TWA are designed to be low-effort,
+even if you've never developed for Android before.
+Here's what the OYO team did:
+* Created an
+   [Android manifest file](https://developer.android.com/guide/topics/manifest/manifest-intro)
+   containing a `DEFAULT_URL` and
+   [intent filters](https://developer.android.com/guide/components/intents-filters)
+   to allow the app to display content from [oyorooms.com](https://oyorooms.com).
+* Removed the browser's URL bar using
+   [Digital Asset Link](https://developers.google.com/digital-asset-links/v1/getting-started)
+   verification.
+* Created a launcher icon.
+* Created a custom splash screen.
+
+And here's the result:
+<figure class="w-figure w-figure--center">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/oyo-case-study/oyo-lite-orig.webm" type="video/webm; codecs=vp8">
+    <source src="https://storage.googleapis.com/web-dev-assets/oyo-case-study/oyo-lite-orig.mp4" type="video/mp4; codecs=h264">
+  </video>
+ <figcaption class="w-figcaption">
+    OYO Lite in action.
+  </figcaption>
+</figure>
+
+For a more detailed walkthrough of building a basic TWA,
+check out Peter McLachlan and Andre Bandarra's
+[TWA talk from Google I/O 2019](https://youtu.be/6lHBw3F4cWs).
+
+To learn more about how the OYO team created OYO Lite—including
+a deep dive into their approach for creating a splash screen
+that's accessible to all Android users—take a look at Ankit Jain's
+[post on Medium](https://medium.com/@ankitjainaj/3dd327d7afc5).
+
+## How a TWA helped OYO meet the needs of its users
+
+By storing most of their app assets in Chrome's cache,
+the OYO team was able to get the initial download size
+for OYO Lite down to a svelte 850&nbsp;KB.
+That's just 7% the size of their native Android app!
+
+That small footprint combined with the amenities of a native app
+downloadable from the Google Play Store led to significant gains
+in user engagement:
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">3<sub class="w-stat__sub">x</sub></p>
+    <p class="w-stat__desc">PWA's conversion rate</p>
```

Comment:
I think the confusion comes from trying to cram necessarily detailed phrases into the stats widget. It was worth trying for the engagement, but since it's confusing, I'm just gonna convert to a list.